apps/aecore/src/aec_db.erl

```diff
@@ -837,6 +856,24 @@ wait_for_tables(Tabs, Sofar, _, _) ->
     init:stop(),
     erlang:error({tables_not_loaded, Tabs}).
 
+%% Prepare bypass
+prepare_mnesia_bypass() ->
+    P = mnesia:table_info(aec_account_state, all),
+    TNames = [N || {N,_} <- tables()],
+    case proplists:get_value(rocksdb_copies, P, []) of
+        [] -> ok;
+        [_] ->
+            HIndex = 'mnesia_ext_proc_aec_headers-4-_ix',
+            {HIndexRef, set} = gen_server:call(HIndex, get_ref),
```

Comment:
Perhaps it would be cleaner to export `mnesia_rocksdb:get_ref(rocksdb_copies,Tab)`, which also by design resolves `mnesia_rocksdb:get_ref(rocksdb_copies, {Tab, index, IxPos})`.

BTW, the index positions (and type of index - with rocksdb, always ordered) can be fetched using `mnesia_lib:val(Tab, index_info)`